src/main/java/org/cryptomator/frontend/fuse/FileAttributesUtil.java

```diff
@@ -78,4 +78,36 @@ public void copyBasicFileAttributesFromNioToFuse(BasicFileAttributes attrs, File
 		}
 	}
 
+	public long posixPermissionsToOctalMode(Set<PosixFilePermission> permissions) {
+		long mode = 0;
+		if (permissions.contains(PosixFilePermission.OWNER_READ)) {
```

Comment:
This is a rare case where I think it is actually more legible to use one-liners and skip the blocks. Just as it is used in [`octalModeToPosixPermissions`](https://github.com/cryptomator/fuse-nio-adapter/blob/1.0.3/src/main/java/org/cryptomator/frontend/fuse/FileAttributesUtil.java#L38-L48).